docs/central-users.rst

```diff
@@ -19,67 +19,94 @@ Central features Role-based User permissioning. In the current release of Centra
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Forms**                                                                                    |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| List All                         | x             | x               | x              | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Edit Attachments                 | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Edit Details                     | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Set State                        | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Delete                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Form Submissions**                                                                         |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Edit & Review                    | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Comment                          | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View & Download                  | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| OData Access                     | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entity Lists**                                                                             |
```

Comment:
I personally don't think the "Project" prefix adds much, but I wanted to be consistent.